docs/html/topics/https-certificates.md

```diff
@@ -8,8 +8,8 @@
 
 By default, pip will perform SSL certificate verification for network
 connections it makes over HTTPS. These serve to prevent man-in-the-middle
-attacks against package downloads. This does not use the system certificate
-store but, instead, uses a bundled CA certificate store from {pypi}`certifi`.
+attacks against package downloads. Pip by default uses a bundled CA certificate
+store from {pypi}`certifi`.
```

Comment:
(Don't know what happened to my comment 🙁)

Is this right? Don't we use the system certs (on 3.10+) and the bundled certs now?

Reply:
My bad, this might have gotten clobbered in the fairly hands-on rebase I had to do to get this branch back in order, let me fix this up! Thank you for catching this.

Reply:
Check out https://github.com/pypa/pip/pull/11647/commits/e137d6e69ffe5c6dcf4342d31535e047511327a9, happy to restructure things if this isn't clear. I'm hoping for most users things will "just work" and the only thing they need to know about is why they're seeing a different behavior than their coworker.